include/openssl/tls1.h

```diff
@@ -868,6 +872,9 @@ SSL_CTX_callback_ctrl(ssl,SSL_CTRL_SET_TLSEXT_TICKET_KEY_CB,(void (*)(void))cb)
 # define TLS1_TXT_DHE_PSK_WITH_CHACHA20_POLY1305           "DHE-PSK-CHACHA20-POLY1305"
 # define TLS1_TXT_RSA_PSK_WITH_CHACHA20_POLY1305           "RSA-PSK-CHACHA20-POLY1305"
 
+/* TLSv1.3 ciphersuites */
+# define TLS1_3_TXT_AES_128_GCM_SHA256                     "TLS_AES_128_GCM_SHA256"
```

Comment:
I don't overly care too much whether its the standard name or not (although I do think standardisation would be a good thing). It needs to be somehow different to the current naming scheme, to disambiguate it from the existing ciphersuites. If we don't like my suggested approach of going with the RFC names, then please make an alternative suggestion.

WRT, decoupling from the current cipherstring processing: that may well be where we end up...but that's for a later stage in the development. Not right now.